app/models/user.rb

```diff
@@ -6,4 +6,8 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   has_many :reviews
+
+  def admin?
+    role == "admin"
+  end
```

Comment:
nice method! this is exactly the sort of clean looking code that you want to be building.